### PR TITLE
{bio}[foss/2022a] genomepy v0.15.0, mygene v3.2.2 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3135.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/e8c32542b412053a3dd290b5082e2aa7 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3135.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/a041f6de86ecae8f31b827cd82791b32 for a full test report.

File: easybuild/easyconfigs/g/genomepy/genomepy-0.15.0-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    ('Biopython', '1.79'),
+    ('mygene', '3.2.2'),
+    ('pyfaidx', '0.7.1'),
+    ('PyYAML', '6.0'),
+    ('protobuf-python', '3.19.4'),
+]
+
+use_pip = True
+
+# avoid hatchling requirement to install infercnvpy
```
Comment (casparvl) [inline easybuild/easyconfigs/g/genomepy/genomepy-0.15.0-foss-2022a.eb:24]: `infercnvpy`? I think this comment may have been copied from another EasyConfig, you probably forgot to change the name here :)

**Review by casparvl — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/genomepy/genomepy-0.15.0-foss-2022a.eb (hunk 1 above)
Comment (casparvl) [reply easybuild/easyconfigs/g/genomepy/genomepy-0.15.0-foss-2022a.eb]: Ok, fixed it myself since it was such a minor change...

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
tcn260.local.snellius.surf.nl - Linux Rocky Linux 8.7, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/a7e6e127bb6ce9fa7b435f3a7b306547 for a full test report.

**Review by casparvl — APPROVED** [review]
Lgtm!

**Comment by casparvl** [comment]
Going in, thanks @boegel!

**Pull request merged**